### ISSUE TITLE
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc9 in position 0: invalid continuation byte

### ISSUE THREAD

**Issue by ProfessorVenkman** [fyookball/electrum#676]
Crash Report
============

This crash report was reported through the automatic crash reporting system 🤖

Traceback
--------------

```Python traceback
  File "/usr/local/Electron-Cash-3.1.6/lib/util.py", line 613, in run_with_except_hook
    run_original(*args2, **kwargs2)
  File "/usr/local/Electron-Cash-3.1.6/lib/interface.py", line 233, in run
    socket = self.get_socket()
  File "/usr/local/Electron-Cash-3.1.6/lib/interface.py", line 139, in get_socket
    s = self.get_simple_socket()
  File "/usr/local/Electron-Cash-3.1.6/lib/interface.py", line 103, in get_simple_socket
    l = socket.getaddrinfo(self.host, self.port, socket.AF_UNSPEC, socket.SOCK_STREAM)
  File "/usr/lib/python3.5/socket.py", line 733, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):

UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc9 in position 0: invalid continuation byte
```

Reporter
------------

This issue was reported by 1 user(s):

| Electrum Version  | Operating System  | Wallet Type  | Locale |
|---|---|---|---|
| 3.1.6  | Linux-4.9.0-6-amd64-x86_64-with-debian-9.4 | standard | fr_FR |

Additional Information
------------------------

The reporting user(s) did not provide additional information.

**Comment by rt121212121** [comment]
https://bugs.python.org/issue26227

This was fixed in Python > 3.5.1, from what I can tell from the cpython bug above.  Either this should be closed as noise, or we should enforce a minimum Python version in the code.

**Comment by SomberNight** [comment]
but the reporter is not on Windows? `Linux-4.9.0-6-amd64-x86_64-with-debian-9.4`

**Comment by rt121212121** [comment]
You're right, I missed that.  There are no fixes for this, for Linux, for 3.5.

**Comment by rt121212121** [comment]
Given that, I'd assume user error and bad data.
